src/constants.js

```diff
@@ -173,6 +173,21 @@ export const TAILWIND_MERGE_EXTENSION = {
       "ring-offset-w": [{ "ring-offset": ["dynamic"] }],
       "ring-offset-color": [{ "ring-offset": ["color-dynamic"] }],
       "font-size": [{ text: ["2xs"] }],
+      roundedSm: [{ rounded: ["dynamic-sm", "inherit"] }],
+      "rounded-s-sm": [{ "rounded-s-sm": ["dynamic-sm", "inherit"] }],
+      "rounded-e-sm": [{ "rounded-e-sm": ["dynamic-sm", "inherit"] }],
+      "rounded-t-sm": [{ "rounded-t-sm": ["dynamic-sm", "inherit"] }],
+      "rounded-r-sm": [{ "rounded-r-sm": ["dynamic-sm", "inherit"] }],
+      "rounded-b-sm": [{ "rounded-b-sm": ["dynamic-sm", "inherit"] }],
+      "rounded-l-sm": [{ "rounded-l-sm": ["dynamic-sm", "inherit"] }],
+      "rounded-ss-sm": [{ "rounded-ss-sm": ["dynamic-sm", "inherit"] }],
+      "rounded-se-sm": [{ "rounded-se-sm": ["dynamic-sm", "inherit"] }],
+      "rounded-ee-sm": [{ "rounded-ee-sm": ["dynamic-sm", "inherit"] }],
+      "rounded-es-sm": [{ "rounded-es-sm": ["dynamic-sm", "inherit"] }],
+      "rounded-tl-sm": [{ "rounded-tl-sm": ["dynamic-sm", "inherit"] }],
+      "rounded-tr-sm": [{ "rounded-tr-sm": ["dynamic-sm", "inherit"] }],
+      "rounded-br-sm": [{ "rounded-br-sm": ["dynamic-sm", "inherit"] }],
+      "rounded-bl-sm": [{ "rounded-bl-sm": ["dynamic-sm", "inherit"] }],
```

Comment:
no its wrong, you don't need to redefine tw default sm and lg classes.

here is the right approach:

```js
rounded: [{ rounded: ["dynamic", "dynamic-sm", "dynamic-lg", "inherit"] }],
... and others same way
```